river/test_estimators.py

```diff
@@ -75,6 +76,8 @@ def iter_estimators_which_can_be_tested():
         imblearn.RandomUnderSampler,
         imblearn.RandomSampler,
         model_selection.SuccessiveHalvingClassifier,
+        neighbors.NearestNeighbors,
+        neighbors.KNNRegressor,
```

Comment:
Any particular reason why `KNNRegressor` can't be tested?

Reply:
I'll add it back - I forget when I originally removed it!